fixtures/webstudio-cloudlfare-template/functions/[[path]].ts

```diff
@@ -0,0 +1,8 @@
+import { createPagesFunctionHandler } from "@remix-run/cloudflare-pages";
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore - the server build file is generated by `remix vite:build`
```

Comment:
should it be committed?

Reply:
It's from remix cloudflare template gonna leave as is